app/views/container_project/_show_dashboard.html.haml

```diff
@@ -86,7 +86,7 @@
               %thead
                 %tr
                   %th Name
-                  %th Phase
+                  %th Status
```

Comment:
This whole haml lacks gettext, fix in https://github.com/ManageIQ/manageiq-ui-classic/pull/2829